### PR TITLE
Feat/teimp duration

### PR DESCRIPTION
**Pull request by mikevespi**
Card: https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/bcgov/cas-cif/763

### CODE REVIEW

File: app/lib/helpers/getDurationFromDates.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { DateTime } from "luxon";
+
+const getDurationFromDates = (startDate: string, endDate: string) => {
```
Comment (matthieu-foucault) [inline app/lib/helpers/getDurationFromDates.ts:3]: Can we have a small comment indicating the format this function returns the duration in?

File: app/lib/theme/widgets/DateWidget.tsx [hunk 2]
```diff
@@ -62,36 +62,44 @@ const DateWidget: React.FC<WidgetProps> = ({
...
       <style jsx>{`
         :global(.react-datepicker__day.react-datepicker__day--keyboard-selected) {
           background: none;
           color: black;
         }
-        div {
+        .dateWidgetWrapper {
+          display: flex;
+          align-items: center;
+          gap: 1rem;
```
Comment (matthieu-foucault) [inline app/lib/theme/widgets/DateWidget.tsx:100]: I didn't know about the `gap` property, that's handy!

File: app/tests/unit/components/Form/ProjectEmissionIntensityReportForm.test.tsx [hunk 3]
```diff
@@ -160,6 +160,8 @@ describe("the emission intensity report form component", () => {
     expect(
       screen.getByLabelText(/Measurement period end date/i)
     ).toHaveTextContent(/Jan[.]? 02, 2023/);
+    expect(screen.getByText(/duration: 12 months/i)).toBeInTheDocument();
+    expect(screen.getByText(/duration: 12 months/i)).toBeInTheDocument();
```
Comment (matthieu-foucault) [inline app/tests/unit/components/Form/ProjectEmissionIntensityReportForm.test.tsx:164]: duplicate line

File: app/tests/unit/lib/helpers/getDurationFromDates.test.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import getDurationFromDates from "lib/helpers/getDurationFromDates";
+
+describe("The getDurationFromDates helper method", () => {
```
Comment (matthieu-foucault) [inline app/tests/unit/lib/helpers/getDurationFromDates.test.ts:3]: can we test the cases where the number or months or days are `0`

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]
just a couple of extra test cases and some comments needed, looks good otherwise

**Review by BCerki — APPROVED** [review]

**Review by matthieu-foucault — APPROVED** [review]

**Review by dleard — APPROVED** [review]

**Pull request merged**